### ISSUE TITLE
EightPointsGuzzleBundle support

### ISSUE THREAD

**Issue by dominikzogg** [auxmoney/OpentracingBundle-Guzzle#9]
Within https://github.com/8p/EightPointsGuzzleBundle the handlers are services by themself. The Client doesn't seem to get registred.

id: eight_points_guzzle.handler_stack.<client>
class: GuzzleHttp\HandlerStack

I would suggest the following change:

```php
if ($definition->getClass() === Client::class) {
    $this->addMiddlewareToHandlerStack($container, $definition, $clientServiceName);
} elseif () {
    $definition->addMethodCall('push', [new Reference(GuzzleRequestSpanning::class)]);
    $definition->addMethodCall('push', [new Reference(GuzzleTracingHeaderInjection::class)]);
}
```

**Comment by cawolf** [comment]
Hi @dominikzogg,
you are right, the condition to add method calls to HandlerStacks was quite tight. Within #8 , we added the `elseif` branch like you suggested and added some functional tests for it.

#8 will be merged as soon as all opentracing repositories are ready to be released as a new major version `0.4`.
Best, @cawolf

**Comment by cawolf** [comment]
Release v0.4.0 contains the fix for this issue.

Thank you for reporting the issue and improving the bundle! :)
